package.json

```diff
@@ -30,25 +30,25 @@
   "dependencies": {
     "dotenv": "^4.0.0",
     "env2": "^2.2.0",
+    "i18n": "^0.8.3",
     "keystone": "^4.0.0-beta.5",
     "keystone-utils": "^0.4.0",
     "pug": "^2.0.0-rc.1",
-    "underscore": "^1.8.3",
-    "tachyons-custom": "^4.5.4"
+    "tachyons-custom": "^4.5.4",
+    "underscore": "^1.8.3"
   },
   "devDependencies": {
+    "autoprefixer": "^6.7.3",
     "eslint": "^3.19.0",
     "keystone-nightwatch-e2e": "^0.2.14",
     "nightwatch": "^0.9.16",
-    "saucelabs": "^1.4.0",
-    "selenium-download": "^2.0.10",
-    "autoprefixer": "^6.7.3",
-    "eslint": "^3.19.0",
     "nodemon": "^1.11.0",
     "postcss-clean": "^1.0.2",
     "postcss-cli": "^2.6.0",
     "postcss-custom-media": "^5.0.1",
     "postcss-custom-properties": "^5.0.2",
-    "postcss-import": "^9.1.0"
+    "postcss-import": "^9.1.0",
```

Comment:
Okay, perhaps add Related #141 too for future reference so we know this PR includes a temp solution to that